data-es6.js

```diff
@@ -272,7 +272,7 @@ exports.browsers = {
   firefox37: {
     full: 'Firefox',
     short: 'FF 37',
-    obsolete: very,
```

Comment:
We should be consistent; and they should probably all have it to keep diffs small.